source/renderer/app/components/wallet/WalletRestoreDialog.js

```diff
@@ -486,10 +569,7 @@ export default class WalletRestoreDialog extends Component<Props, State> {
           }
           placeholder={
             !this.isCertificate()
-              ? intl.formatMessage(messages.recoveryPhraseInputHint, {
-                  numberOfWords:
-                    walletType === WALLET_RESTORE_TYPES.LEGACY ? '12' : '15',
-                })
+              ? intl.formatMessage(messages.recoveryPhraseInputHint)
               : intl.formatMessage(messages.shieldedRecoveryPhraseInputHint, {
                   numberOfWords: 27,
```

Comment:
@DeeJayElly please replace `27` hardcoded value with the `PAPER_WALLET_RECOVERY_PHRASE_WORD_COUNT `.

Reply:
Done